tests/fixed/semicolon_spacing.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+echo -1;
+
+echo 'foo';
+
+
+$qb->select()
+    ->from()
+    ->where();
```

Comment:
EOL is missing

Reply:
Isn't that what CS is complaining, but I also couldn't identified what is :(